contributing/style_guide.md

```diff
@@ -185,9 +185,9 @@ export function pwrite(options: PWrite) {}
 
 ### Export all interfaces that are used as parameters to an exported member
 
-Whenever you are using interfaces that are included in the arguments of an
-exported member, you should export the interface that is used. Here is an
-example:
+Whenever you are using interfaces that are included in the parameters or return
```

Comment:
I'm changing the overall meaning of this line. Let me know if my assumptions about its meaning are incorrect.